src/app/components/UploadImage/UploadImage.tsx

```diff
@@ -13,9 +13,11 @@ function UploadImage({ onUpload }: ImageUploadProps): JSX.Element {
     }
     const file = event.target.files[0];
 
+    const key = import.meta.env.VITE_CLOUDINARY_UPLOAD as string;
+
     const formData = new FormData();
     formData.append('file', file);
-    formData.append('upload_preset', 'bmnorbj6');
```

Comment:
Is this still your actual cloudinary code? If so, you should generate a new one and put it in VITE_CLOUDINARY_UPLOAD. Otherwise your code could still be seen by other people.

Reply:
I changed it before hiding